PowerEditor/visual.net/notepadPlus.vcxproj

```diff
@@ -301,7 +301,6 @@ copy ..\src\contextMenu.xml ..\bin64\contextMenu.xml
     <ClCompile Include="..\src\ScitillaComponent\DocTabView.cpp" />
     <ClCompile Include="..\src\WinControls\DocumentMap\documentMap.cpp" />
     <ClCompile Include="..\src\EncodingMapper.cpp" />
-    <ClCompile Include="..\src\WinControls\OpenSaveFileDialog\FileDialog.cpp" />

```

Comment:
Shouldn't we add `..\src\WinControls\OpenSaveFileDialog\CustomFileDialog.cpp`?

Reply:
It is already there, see https://github.com/notepad-plus-plus/notepad-plus-plus/blob/611790310c89e1bf1357b1be7a9fdfd3abbebd68/PowerEditor/visual.net/notepadPlus.vcxproj#L295